packages/block-library/src/template-part/edit/placeholder/index.js

```diff
@@ -13,71 +13,91 @@ import { store as coreStore } from '@wordpress/core-data';
  * Internal dependencies
  */
 import TemplatePartSelection from '../selection';
+import PatternsSetup from './patterns-setup';
 
 export default function TemplatePartPlaceholder( {
 	area,
+	clientId,
 	setAttributes,
-	innerBlocks,
```

Comment:
This must be a leftover 👍 . This was always empty array.

Reply:
Yeah, I think its a leftover from the old implementation of the 'make template part' flow (selecting existing blocks => ellipsis menu).